docs/boshdirector.adoc

```diff
@@ -28,114 +40,307 @@ Click on the zone; you'll notice an NS-type entry for the subdomain, as shown be
 
-== Post-Installation
+. **Ask your instructor to validate your settings.**
 
-. in the Ops Manager, select the GCP _tile_.  We need to retrieve some information:
-.. the Ops Manager IP address: navigate to the _Status_ tab, note the IP address for the Ops Manager Director
-.. navigate to the _Credentials_ tab, click the link to the director credentials, note the identity and password displayed in the json response.
+. Browse to the `Installation Dashboard`.  Then click `Apply Changes` to create an instance of Ops Manager Director.
 
-. verify that you can ssh into the Ops Manager with:
 +
-[source.terminal]
-----
-gcloud compute ssh ubuntu@{{env_name}}-ops-manager
-----
+[.thumbs]
+.Apply Changes
+image::director-apply-changes-gcp.png[Apply Changes]
+
 +
-NOTE: The above command dynamically creates a key pair for logging in to the VM
+The installation should take about 10 minutes. You can view the log details as the installation is happening (`Show verbose output`).
+
+. When the installation is successful, you should have receive a message saying "Changes Applied...".
+
 +
-The bosh client is pre-installed on Ops Manager VM.  For security, the bosh director VM is usually not made accessible from the outside world.  The Ops Manager VM serves as a bastion VM from which we can communicate with the bosh director.
+[.thumbs]
+.Changes Applied
+image::director-success-gcp.png[Changes Applied]
 
-. Target, and login to the bosh director:
 +
-[source.terminal]
-----
-bosh --ca-cert /var/tempest/workspaces/default/root_ca_certificate target {{director_ip_address}}
-----
+You can ignore the message about exporting a backup of the installation for now.  That will be the topic of another lab.  Just keep in mind that it is good to do after every deployment.
+
+. When the installation is complete, click on `Changelog` to view the installation logs.
+
 +
-When prompted to log in, enter `director` for the user name, and the previously noted secret for the password.
+[.thumbs]
+.Change Log
+image::change-log-gcp.png[Change Log]
+
+. Explore the `Ops Manager Director` tile.  Review the `Status` and `Credentials` tabs.
 
-. Invoke the following command:
 +
-[source.terminal]
-----
-bosh deployments
-----
+[.thumbs]
+.Director Status
+image::director-status-gcp.png[Director Status]
+
 +
-The response should be empty at the moment, as we have not yet used the bosh director to deploy anything.
+[.thumbs]
+.Credentials
+image::director-credentials-gcp.png[Credentials]
 
-You may now log out of the Ops Manager.
++
+In the Google Cloud Console, visit the _Compute Engine -> VM Instances_ section and observe that a new VM will have been created for the Ops Manager (BOSH) Director.
 
-Congratulations, you have verified the proper functioning of the bosh director and are now ready to move on to installing the elastic runtime (our next lab).
+**Congratulations!** You have successfully installed the Ops Manager Director.
```

Comment:
We have to be careful with referring to other modules, given that each module should stand on it's own so it can be included as part of any course offering.